lib/fetch/body.js

```diff
@@ -172,17 +181,10 @@ function extractBody (object, keepalive = false) {
 
     stream =
       object instanceof ReadableStream ? object : ReadableStreamFrom(object)
-  } else {
-    // TODO: byte sequence?
-    // TODO: scalar value string?
-    // TODO: else?
-    source = toUSVString(object)
-    type = 'text/plain;charset=UTF-8'
```

Comment:
What if none of the above is true? assert(false)?

Reply:
I think the spec lists all the potential values it can be. If the `body` passed isn't one of the allowed values, webidl converts the value to a USVString for us.

Reply:
See https://github.com/nodejs/undici/blob/d23f5dd00fed2d6fb85baeef9b714f3a0d1bad88/lib/fetch/response.js#L524-L578